.gitmodules

```diff
@@ -1,6 +0,0 @@
-[submodule "osmosis"]
-	path = osmosis
-	url = https://github.com/openstreetmap/osmosis
-[submodule "protobuf"]
-	path = protobuf
-	url = https://github.com/google/protobuf
```

Comment:
@MorbZ Can you confirm this can be removed? For developing / building the project those are indeed not necessary since `osmosis-osm-binary` is included as a maven-central dependency, and `protobuf-java` is not used by the project itself.